Disable select options in dropdown (#953)
vtex-apps/store-components: CHANGELOG.md

> ## [Unreleased]
> 
> ### Added
> - Add `disableUnavailableSelectOptions` prop for `SKUSelector` in order to disable unavailable items for `displayMode` select

Reviewer: Just to make it easier, when you add an entry to the CHANGELOG, add it right below the `[Unreleased]` header, without an empty line. This will avoid conflicts :)

```suggestion
## [Unreleased]
### Added
- `disableUnavailableSelectOptions` prop to `SKUSelector` in order to disable unavailable items when `displayMode` is set to `select`.
```
